app/src/main/res/layout/item_selection_interaction_items.xml

```diff
@@ -14,14 +14,14 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:minHeight="48dp"
-    android:layout_margin="4dp"
+    android:paddingTop="8dp"
```

Comment:
We can reformat this file.

Reply:
Yes, I avoided this earlier just to keep PR review easy. Will now make a commit to fix all reformatting.